service/controller/v1/drainer_resource_set.go

```diff
@@ -95,46 +91,5 @@ func NewDrainerResourceSet(config DrainerResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	handlesFunc := func(obj interface{}) bool {
-		drainerConfig, err := key.ToDrainerConfig(obj)
-		if err != nil {
-			return false
-		}
-
-		if key.VersionBundleVersionFromDrainerConfig(drainerConfig) == VersionBundle().Version {
-			return true
-		}
-
-		return false
-	}
-
-	initCtxFunc := func(ctx context.Context, obj interface{}) (context.Context, error) {
-		drainerConfig, err := key.ToDrainerConfig(obj)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-
-		lm := loggermeta.New()
-		lm.KeyVals["cluster"] = key.ClusterIDFromDrainerConfig(drainerConfig)
-		lm.KeyVals["node"] = key.NodeNameFromDrainerConfig(drainerConfig)
-
-		return loggermeta.NewContext(ctx, lm), nil
-	}
```

Comment:
I'm not sure how this functionality fits into the new `operatorkit` standard. Particularly the `initCtxFunc` behavior -- is this no longer necessary at all, or do I need to propagate this downward through some other means?